plugins/synced_folders/rsync/helper.rb

```diff
@@ -63,7 +63,11 @@ def self.rsync_single(machine, ssh_info, opts)
         opts[:owner] ||= ssh_info[:username]
         opts[:group] ||= ssh_info[:username]
 
+        # set log level
+        log_level = ssh_info[:log_level] || "FATAL"
+
```

Comment:
This `log_level` looks to be extraneous here, or was logging level meant to be set within the options?